tests/errmsgs/t8654_2.nim

```diff
@@ -0,0 +1,7 @@
+discard """
+  errormsg: "cannot have typedesc as const value, use 'type a = int' instead"
+  line: 6
+"""
+
+const a: typedesc = typedesc[int]
+echo a is typedesc[int]
```

Comment:
if you're gonna error on line 6, why have line 7? (+ same in other files)

Reply:
Sanity check? I copied existing tests

Reply:
why is that a sanity check, if it won't even get compiled?
seems like you could put whatever there instead...
maybe existing tests could be updated too